ddtrace/contrib/internal/ray/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""
+The ray integration traces:
+ - Job lifetime (job submit, job run)
+ - Task submission and execution
+ - Actor method submission and execution
+
+Enabling
+~~~~~~~~
+
+The Ray integration is enabled automatically when you use
+:ref:`ddtrace-run<ddtracerun>` or :ref:`import ddtrace.auto<ddtraceauto>`.
+
+Or use :func:`patch() <ddtrace.patch>` to manually enable the Ray integration::
+
+    from ddtrace import patch
+    patch(ray=True)
+
+The recommended way to instrument Ray, is to instrument the Ray cluster.
+You can do it by starting the Ray head with a tracing startup hook::
+
+    ray start --head --tracing-startup-hook=ddtrace.contrib.ray:setup_tracing
+
+
+Configuration
+~~~~~~~~~~~~~
+
+The Ray integration can be configured using environment variables:
+
+- ``DD_TRACE_RAY_RESUBMIT_LONG_RUNNING_INTERVAL``: Interval for resubmitting long-running
```

Comment:
Can we make this a generic config something like DD_TRACE_LONG_RUNNING_SPAN_SUBMISSION_InTERVAL. We can say it is only supported for Ray spans for now and then expand support in a future PR.

Reply:
I updated the name to be consistent with the java name !